tool/grind.dart

```diff
@@ -3,24 +3,120 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:convert';
 import 'dart:io' hide ProcessException;
 
-import 'package:dartdoc/dartdoc.dart' show defaultOutDir;
 import 'package:dartdoc/src/io_utils.dart';
 import 'package:grinder/grinder.dart';
-import 'package:html/dom.dart';
-import 'package:html/parser.dart' show parse;
 import 'package:path/path.dart' as path;
 import 'package:yaml/yaml.dart' as yaml;
 
 main([List<String> args]) => grind(args);
 
-final Directory docsDir =
-    new Directory(path.join('${Directory.systemTemp.path}', defaultOutDir));
+final Directory dartdocDocsDir = Directory.systemTemp.createTempSync('dartdoc');
```

Comment:
This script creates these temp dirs on every run, whether they're used or not. They're in temp, so it may not actually matter, but consider not creating them until the variables are first referenced?

Reply:
Done.